torch/fx/passes/runtime_assert.py

```diff
@@ -204,6 +187,8 @@ def add_runtime_asserts(ras):
                 )
                 # if we've already added a constrain_range call for this symbol,
                 # then single-symbol bound asserts like u0 >= 0, u0 <= 5 are redundant.
+                or _has_unsupported_sympy_function(ra.expr)
```

Comment:
Can you move comments on each of these cases before the line implementing them? Otherwise I was reading the wrong comment for this line lol.